setup.cfg

```diff
@@ -5,3 +5,13 @@ versionfile_source = montblanc/_version.py
 versionfile_build = montblanc/_version.py
 tag_prefix =
 parentdir_prefix = montblanc-
+
+[build_ext]
+# Warning this marches code natively to the processor architecture
+# which will make any binary distributed with the wheel non-portable
+# between systems. Ensure to disable prior to building wheels and/or
+# images
+march_native=False
+# further flags to pass onto the compiler and linkers
+compiler_verbosity=-Wno-ignored-attributes -Wno-deprecated-declarations -Wno-unused-variable
+linker_options=
```

Comment:
Montblanc is always compiled from source, so this makes sense to me.

Reply:
you happy with this? - it is necessary to control especially the marching options for building docker images

Reply:
Yep, I don't think its likely that we'll ever build a wheel.